examples/marketing/banner

```diff
@@ -0,0 +1,44 @@
+require 'bigcommerce'
+
+Bigcommerce.configure do |config|
+  config.auth = 'legacy'
+  # You will get this url when registering for an API key
+  config.url = ENV.fetch['BC_STORE_HASH']
+  config.username = ENV.fetch['BC_CLIENT_ID']
+  config.api_key = ENV.fetch['BC_ACCESS_TOKEN']
+end
+
+puts Bigcommerce::System.time
+puts Bigcommerce.api_limit
+
+# List banners
+@banner = Bigcommerce::Banner.all
+puts @banner
+
+# Get a banner
+puts Bigcommerce::Banner.find(@banner[0].id)
+
+# Get a count of banners
+puts Bigcommerce::Banner.count
+
+# Create a banner
+@banner = Bigcommerce::Banner.create(
+	name: "Fourth of July Banner",
+	content: "<div id = "holidayBanner"><img src = "/FourthOfJuly.png" /></div>",
```

Comment:
This looks as though you're breaking the string on the second double quote so the string would look like `<div id =` because the second double quote terminates the string.

I'd change it to `'<div id="holidayBanner"><img src="/FourthOfJuly.png"/></div>'`